src/components/authortimeline/AuthorTimeline.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import { Timeline, TimelineItem } from 'vertical-timeline-component-for-react';
+
+const inputdata = [
```

Comment:
I would rather write "inputData" using camelcase, but as soon as it is a draft constant and will be removed, do not think that it needs refactoring.